browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -87,6 +88,11 @@ int OnBeforeURLRequest_StaticRedirectWorkForGURL(
   static URLPattern widevine_google_dl_pattern(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS,
       kWidevineGoogleDlPrefix);
+#if BUILDFLAG(WIDEVINE_ARM64_DLL_FIX)
+  static URLPattern widevine_google_dl_pattern_win_arm64(
+      URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS,
```

Comment:
Let's stick to HTTPS only. We shouldn't need to fall back to HTTP for this and we probably don't want to anyways because I don't think we're checking the signature on the downloaded files.